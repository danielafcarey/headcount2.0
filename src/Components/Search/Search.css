@font-face {
    font-family: placeHolder;
    src: url(../../fonts/Crackvetica.ttf);
}

.search-container{
  margin: 0 auto;
  width: 40%;

}

.search{
  width: 100%;
  height: 40px;
  font-size: x-large;
  border: none;
  border-left: 6px solid purple;
}

.search::placeholder {
  font-size: x-large;
  padding-left: 10px;
  font-family: 'Bangers', cursive;
  letter-spacing: 2px

}

.search-border:focus {
    border: 3px solid #555;
}

 @media only screen and (max-width: 600px) {
    .search {
      width: 80%;
    }
}