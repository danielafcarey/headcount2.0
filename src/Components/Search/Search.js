import React from 'react';
import PropTypes from 'prop-types';
import './Search.css';

const Search = ({ updateRepoInState }) => {
  return (
    <div className="search-container">
      <input 
        placeholder="Search for a district..."
        className="search search-border"
        onChange={ (event) => updateRepoInState(event.target.value) }
      />
    </div>
  );
};

Search.propTypes = {
  updateRepoInState: PropTypes.func.isRequired
};

export default Search;
