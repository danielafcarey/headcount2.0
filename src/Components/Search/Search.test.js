import React from 'react';
import { shallow, mount } from 'enzyme';
import Search from './search';
import PropTypes from 'prop-types';
import renderer from 'react-test-renderer';

describe('Search', () => {

  it('matches the snapshot', () => {
    const searchSnapshot = renderer.create(<Search updateRepoInState={ jest.fn() } />);
    
    expect(searchSnapshot).toMatchSnapshot();
  })
})
