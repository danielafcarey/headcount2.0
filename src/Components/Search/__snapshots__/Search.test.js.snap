// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search matches the snapshot 1`] = `
<div
  className="search-container"
>
  <input
    className="search"
    onChange={[Function]}
    placeholder="Search for a district..."
  />
</div>
`;
